src/main/resources/assets/computercraft/lua/rom/apis/colors

```diff
@@ -25,13 +25,23 @@ function combine( ... )
 end
 
 function subtract( colors, ... )
-	local r = colors
-	for n,c in ipairs( { ... } ) do
-		r = bit32.band(r, bit32.bnot(c))
-	end
-	return r
+    local r = colors
+    for n,c in ipairs( { ... } ) do
+        r = bit32.band(r, bit32.bnot(c))
+    end
+    return r
 end
 
 function test( colors, color )
     return ((bit32.band(colors, color)) == color)
 end
+
+function rgb8( r, g, b )
```

Comment:
It does the following:
```Lua
colours.rgb8(255, 0, 255) -- -> 1.0, 0.0, 1.0
colours.rgb8(0x00FFFF) -- -> 0.0, 1.0, 1.0
```
I guess the latter overload isn't needed anymore since you can pass hex rgb into `term.setColour` directly now.